tcf_website/templates/landing/landing.html

```diff
@@ -1,23 +1,267 @@
+          <a class="dropdown-item" href="{% url 'logout' %}">Logout</a>
+        </div>
+      </div>
+      {% else %}
+      <div class="nav-item">
+        <a class="nav-link" href="{% url 'login' %}">Sign In</a>
+      </div>
+    {% endif %}
+    </div>
+  </div>
+</nav>
+
+<!-- Masthead -->
+<header class="masthead bg-light text-center">
+  <div class="container">
+    <div class="row">
+      <div class="col-xl-9 mx-auto">
+        <h1 class="mb-2">theCourseForum</h1>
+        <p class="lead mb-5">A UVA student community, connecting you to the courses you love.</p>
+      </div>
+      <div class="col-md-10 col-lg-8 col-xl-7 mx-auto">
+        {% include "base/searchbar.html" %}
+      </div>
+    </div>
+  </div>
+</header>
+
+<!-- Icons Grid -->
+<section class="features-icons bg-light text-center">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-4">
+        <a href="{% url 'browse' %}" class="link-unstyled">
+          <div class="card m-3">
+            <div class="card-body features-icons-item mx-auto mb-2">
+              <div class="features-icons-icon d-flex">
+                <i class="icon-list m-auto text-tcf-indigo"></i>
+              </div>
+              <h4 class="card-title">Browse Courses</h4>
+              <p class="card-text font-weight-light mb-0">See what other UVA students say about classes and professors</p>
+            </div>
+          </div>
+        </a>
+      </div>
+
+      <div class="col-lg-4">
+        <a href="{% url 'browse' %}" class="link-unstyled">
+          <div class="card m-3">
+            <div class="card-body features-icons-item mx-auto mb-2">
+              <div class="features-icons-icon d-flex">
+                <i class="icon-chart m-auto text-tcf-indigo"></i>
+              </div>
+              <h4 class="card-title">View Grades</h4>
+              <p class="card-text font-weight-light mb-0">Check out actual grade distributions from past semesters</p>
+            </div>
+          </div>
+        </a>
+      </div>
+
+      <div class="col-lg-4">
+        <a href="{% url 'new_review' %}" class="link-unstyled">
+          <div class="card m-3">
+            <div class="card-body features-icons-item mx-auto mb-2">
+              <div class="features-icons-icon d-flex">
+                <i class="icon-pencil m-auto text-tcf-indigo"></i>
+              </div>
+              <h4 class="card-title">Write a Review</h4>
+              <p class="card-text font-weight-light mb-0">Help our community by telling others about your class experience</p>
+            </div>
+          </div>
+        </a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!-- About Team -->
+<section class="about-team bg-light">
+  <div class="row about-header">
+    <div class="col-lg-8 p-4 bg-tcf-indigo text-white text-right">
+      <h3 class="font-weight-light px-4">We'll connect you to courses you'll love, and make enrollment stress free.</h3>
```

Comment:
Not really a technical issue but the wording is a bit clunky (especially when "connecting you to..." is already the subtitle of the center text); anyone have any ideas?

Reply:
Maybe a simpler tagline should go under the `theCourseForum` heading, and then the "uva student community" part should go in the blue banner (because it connects more to "About us")